webpages/styles/components/buttons.css

```
@@ -2,10 +2,10 @@
   color: var(--content-text);
   background: var(--button-background);
   border: 1px solid var(--control-border);
-  padding: 0 12px;
+  display: flex;
```

This makes the preview and text incorrectly aligned unless `vertical-align` is set to `center`.

---

True, I didn't notice.
```suggestion
  display: flex;
  align-items: center;
```